subprojects/docs/src/docs/release/notes.md

```diff
@@ -21,9 +21,35 @@ With [dependency constraints](userguide/managing_transitive_dependencies.html#se
 
 In the example, the version of `commons-codec` that is brought in transitively is `1.9`. With the constraint, we express that we need at lease `1.11` and Gradle will now pick that version during dependency resolution.
 
-### Advanced POM support (preview)
+### Improved POM support: optional dependencies
 
-Gradle now supports [additional features for modules with POM metadata](https://github.com/gradle/gradle/blob/master/subprojects/dependency-management/preview-features.adoc). This includes support for _optional dependencies_, _BOM import_ and _compile/runtime scope separation_. 
+Gradle now creates a dependency constraint for each dependency declaration in a POM file with `<optional>true</optional>`. This constraint will produce the expected result for an optional dependency: if the dependency module is brought in by another, non-optional, dependency declaration, then the constraint will apply when choosing the version for that dependency (e.g., if the optional dependency defines a higher version, that one is chosen).
+
+_Note:_ This is a _Gradle 5.0 feature preview_, which means it is a potentially breaking change that will be activated by default in Gradle 5.0. It can be turned on in Gradle 4.6+ by setting `org.gradle.advancedpomsupport=true` in _gradle.properties_.
+
+### Improved POM support: BOM import
+
+Gradle now [provides support](userguide/managing_transitive_dependencies.html#sec:bom_import) for importing [bill of materials (BOM) files](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies), which are effectively `.pom` files that use `<dependencyManagement>` to control the dependency versions of direct and transitive dependencies. It works by declaring a dependency on a BOM.
+
+    dependencies {
```

Comment:
Is the BOM automatically resolved when declaring a module or does a user need to directly point to it? It's not clear from the example. I am assuming the first dependency declaration _is_ the BOM. Might be worth adding an inline comment in the code.

Reply:
Can we make the claim that you can now switch to the built-in BOM support instead to Pivotal's dependency management plugin?

Reply:
I added comments to the example.

> Can we make the claim that you can now switch to the built-in BOM support instead to Pivotal's dependency management plugin?

I think we can't (yet). There are features in the plugin we do not support. See this discussion: https://github.com/gradle/gradle/issues/1162#issuecomment-343976701

Maybe we can claim that for 5.0 when this is turned on by default and maybe further extended.